### PR TITLE
Add larger image preview for Google Discover

### PR DESCRIPTION
**Pull request by nitro-marky**
## What does this change?

Adding this meta tag allows Google Discover to include larger article previews where the main image is >=1200px. 

[https://developers.google.com/search/docs/advanced/mobile/google-discover?hl=en&visit_id=637424198370039526-3805703503&rd=1](url)

This is already implemented on frontend so this essentially brings DCR to parity.

### CODE REVIEW

**Review by shtukas — APPROVED** [review]

**Review by liywjl — APPROVED** [review]
👍

File: src/web/server/htmlTemplate.ts [hunk 1]
```diff
@@ -68,6 +68,10 @@ export const htmlTemplate = ({
 
     const twitterMetaTags = generateMetaTags(twitterData, 'name');
 
+    // This tag enables pages to be featured in Google Discover as large previews
+    // See: https://developers.google.com/search/docs/advanced/mobile/google-discover?hl=en&visit_id=637424198370039526-3805703503&rd=1
```
Comment (oliverlloyd) [inline src/web/server/htmlTemplate.ts:72]: Useful description!

File: src/web/server/htmlTemplate.ts [hunk 2]
```diff
@@ -68,6 +68,10 @@ export const htmlTemplate = ({
...
+    const googleDiscoverMetaTag = '<meta name="robots" content="max-image-preview:large">';
```
Comment (oliverlloyd) [inline src/web/server/htmlTemplate.ts:73]: We could just inline the string rather than assigning it to a variable?

**Review by oliverlloyd — APPROVED** [review]
Just a comment on maybe not using a var?

I had a quick chat with @gtrufitt about this. Seems 👍

File: src/web/server/htmlTemplate.ts (hunk 2 above)
Comment (nitro-marky) [reply src/web/server/htmlTemplate.ts:73]: No problem, I was just following the style there, but yeah it'll be fine as a string.

**Pull request merged**